### ISSUE TITLE
Hover on yellow/green buttons doesn't pass WCAG. Change to our default black for text/arrow icon

### ISSUE THREAD

**Issue by Erioldoesdesign** [foss-responders/fossresponders.com#96]
In PR #95 the hover colour on buttons for text/arrow icon is the same colour as the button bg/outline e.g. 

Non-hover = Yellow bg with white text and arrow icon
Hover  = Yellow outline with yellow text and arrow icon.

For the #FFDD57 (yellow) and #7BC9AC (green) colours the text and arrow icon when hovering should be #000000 (black) as this then passes colour contrast accessibility :)

Screenshot from the large Adobe XD design file:

![Screenshot 2020-05-13 at 13 53 06](https://user-images.githubusercontent.com/11681324/81815635-21664a80-9522-11ea-9cee-14cf0576aea3.png)

Which I very foolishly left out of the original issue 🤦 my bad

and a link to the AdobeXD design system mocks: https://xd.adobe.com/view/8e9576cb-18da-4d9f-58b8-7d00d753377f-f8a4/screen/890b4715-943c-44a8-ac36-9cc4a4cc1ca1/Design-System-1?fullscreen

**Comment by awelllle** [comment]
I'll finish this up

**Comment by RichardLitt** [comment]
Thank you, @awelllle!
